packages/vue-client/src/components/boards/BadukBoardAbstract.vue

```diff
@@ -25,6 +26,18 @@ const emit = defineEmits<{
 function intersectionClicked(identifier: number) {
   emit("move", identifier);
 }
+
+const viewBox = computed(() => {
+  const xPositions = props.gamestate.board.Intersections.map(
+    (i) => i.Position.X
+  );
+  const yPositions = props.gamestate.board.Intersections.map(
+    (i) => i.Position.Y
+  );
+  return `${Math.min(...xPositions) - 1} ${Math.min(...yPositions) - 1} ${
```

Comment:
At the end of constructing the intersections of the polygonal board, all positions are shifted such that Math.min(...xPositions) and Math.min(...yPositions) are 0, 0. 

However I like that this is handled in the view, so the board is always pictured nicely. So should we remove the abovementioned shifting in the construction algorithm?

Reply:
Interesting. It is somewhat redundant, but I don't mind having both, a visualization that can handle negative coordinates and coordinates that are all non-negative.